File: bronze/2920/c++/main.cpp
#include <iostream>

bool comparison(int *array, int target)
{

    for (int i = 0; i < 8; i++) {
        if (target == 0) {
            if (array[i] != i +1) {
                return false;
            }
        } else {
            if (array[i] != 8 - i) {
                return false;
            }
        }
    }
    return true;
}


int main()
{
    int array[8] = {0};
    for (int i = 0; i < 8; i++) {
        std::cin >> array[i];
    }

    if (comparison(array, 0)) {
        std::cout << "ascending";
    } else if (comparison(array, 1)) {
        std::cout << "descending";
    } else {
        std::cout << "mixed";
    }

    return 0;
}

File: bronze/2920/python/main.py

if (__name__ == "__main__"):
    array = list(map(int, input().split()))
    if (array == [1, 2, 3, 4, 5, 6, 7, 8]):
        print("ascending")
    elif (array == [8, 7, 6, 5, 4, 3, 2, 1]):
        print("descending")
    else:
        print("mixed")